keps/sig-network/563-dual-stack/README.md

```diff
@@ -1416,36 +1410,21 @@ fields of API types, flags, etc.?**
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  Dual stack networking is a functional addition, not a service with SLIs.
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  At a high level, this usually will be in the form of "high percentile of SLI
-  per day <= X". It's impossible to provide comprehensive guidance, but at the very
-  high level (needs more precise definitions) those may be things like:
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99,9% of /health requests per day finish with 200 code
+  N/A
 
 * **Are there any missing metrics that would be useful to have to improve observability 
 of this feature?**
-  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-  implementation difficulties, etc.).
+  N/A
+
+  [TODO] What about crashlooping containers?
+  Since this feature potential has an impact on the runtime environment of the workloads, plus on the IP family returned by DNS lookups (whichever comes back first in some cases), I could foresee some workloads having trouble. Operators should be prepared to monitor for that.
```

Comment:
This line came from a longer feedback section. Please clarify this case.

Reply:
Full context:
 _What about crashlooping containers? I guess I am worried about workloads that suddenly see themselves as dual stack no longer functioning. I realize there is discussion about how that is CNI dependent, but I think as an operator I may want to watch for that in case I have legacy workloads that get confused. The operator wants to know both that the control plane is working fine, but they also want to make sure that this upgrade didn't make a bunch of workloads start failing. Since this feature potential has an impact on the runtime environment of the workloads, plus on the IP family returned by DNS lookups (whichever comes back first in some cases), I could foresee some workloads having trouble. Operators should be prepared to monitor for that._

Reply:
For services: 
1. Services are not in the path of pod creation. Thus malfunctioning bugs in services will not affect pods.
2. Services/Endpoint selection is not in path of pod creation. It runs in kube-controller-manager, this this is N/A

For pods:
1. dualstack components are not in path of pod creation. It is in the path of reporting pod ips. So pod creation will not be affected, if it does then it is a CNI issue.
2. dualstack components are in the path of PodIPs reporting which affects kubelet. If there is a problem (persistent problems) then disabling the featuregate on api-server should solve this.